Sources/PointFree/Account/Account.swift

```diff
@@ -1066,7 +1129,6 @@ private struct AccountData {
   }
 
   var isTeamSubscription: Bool {
-    guard let id = self.stripeSubscription?.plan.id else { return false }
-    return id == .teamMonthly || id == .teamYearly
+    return (self.stripeSubscription?.quantity ?? 0) > 1
```

Comment:
is this the right logic for this?

Reply:
also I made a trello card for us to remove references to `.teamMonthly` and `.teamYearly`

Reply:
this looks right to me